cob_bringup/components/cob4_base.launch

```diff
@@ -3,6 +3,7 @@
 	<arg name="robot"/>
```

Comment:
> can we not keep the name canopen_base.launch?

No. This is semantic naming.
I am aware of the raw3-5 PR, it will get a separate launch file (as it is a different component).
I guess we will introduce a helper file underneath. This one might be called canopen_base.xml ;) 
The first PR that got merged wins. (as usual)